x-pack/plugins/index_lifecycle_management/public/store/reducers/policies.js

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+
+
+import { handleActions } from 'redux-actions';
+import {
+  fetchedPolicies,
+  setSelectedPolicy,
+  setSelectedPolicyName,
+  setSaveAsNewPolicy,
+  setPhaseData
+} from '../actions';
+import { policyFromES } from '../selectors';
+import {
+  PHASE_HOT,
+  PHASE_WARM,
+  PHASE_COLD,
+  PHASE_DELETE,
+  PHASE_ATTRIBUTES_THAT_ARE_NUMBERS,
+  PHASE_ENABLED,
+  PHASE_ROLLOVER_ENABLED,
+  PHASE_FORCE_MERGE_SEGMENTS,
+  PHASE_FORCE_MERGE_ENABLED,
+  PHASE_ROLLOVER_AFTER,
+  PHASE_NODE_ATTRS,
+  PHASE_PRIMARY_SHARD_COUNT,
+  PHASE_REPLICA_COUNT,
+  PHASE_ROLLOVER_AFTER_UNITS,
+  PHASE_ROLLOVER_MAX_AGE,
+  PHASE_ROLLOVER_MAX_AGE_UNITS,
+  PHASE_ROLLOVER_MAX_SIZE_STORED,
+  PHASE_ROLLOVER_MAX_SIZE_STORED_UNITS,
+  PHASE_ROLLOVER_ALIAS,
+  PHASE_ROLLOVER_MAX_DOC_SIZE,
+  PHASE_SHRINK_ENABLED,
+  WARM_PHASE_ON_ROLLOVER
+} from '../constants';
+
+export const defaultWarmPhase = {
```

Comment:
Can we extract these default configurations into a location that's commonly accessible to both the reducers and the selectors? It seems odd for selectors to reach into reducers for these. I'd suggest a `services` directory that's a sibling to `store`.

Reply:
done